ghwatch: Notification reason priority sorting (#84)
bitcoin-core/bitcoin-maintainer-tools: README.md

To keep this repository up to date you can set the interval in seconds in 'auto_update', default is 0 (i.e. no automatic update). Be aware that [bitcoin-gh-meta](https://github.com/zw/bitcoin-gh-meta) is being refreshed every two hours (7200 seconds).

> To enable or disable sorting of the notifications by notification reason (and then by time) you can respectively set 'sort_notifications' to respectively true or false. Default is false.

Reviewer: ```suggestion
Sorting the notifications by {reason, time} can be enabled with the 'sort_notifications' boolean field (default=false).
```
